auth/src/main/java/org/aerogear/mobile/security/AsyncSecurityCheckExecutor.java

```diff
@@ -129,7 +129,7 @@ public AsyncSecurityCheckExecutor build() {
         final AtomicInteger count = new AtomicInteger(checks.size());
 
         for (final SecurityCheck check : checks) {
-            res.put(check.getName(), (executorService.submit(() -> {
+            res.put(check.getDisplayName(), (executorService.submit(() -> {
```

Comment:
Use getType instead of getDisplayName

Reply:
+1. The display name is not guaranteed to be unique, but type is.